src/value-converters/README.md

```diff
@@ -37,10 +39,11 @@ import {HTMLSanitizer} from "aurelia-templating-resources";
 aurelia.use.singleton(HTMLSanitizer, SanitizeHtmlHtmlSanitizer);
 
 const htmlSanitizer = aurelia.container.get(HTMLSanitizer) as SanitizeHtmlHtmlSanitizer;
-htmlSanitizer.escapeAllTheThings()                              // shortcut to allow nothing
-htmlSanitizer.options.disallowedTagsMode = "discard"            // discard tags (rather than escape them)
-htmlSanitizer.options.allowedTags = ['b', 'i', 'img']           // set whitelist (use .concat to add to)
-htmlSanitizer.options.allowedAttributes['img'] = ['src', 'alt'] // whitelist attributes of a tag (does nothing, if tag not whitelisted)
+htmlSanitizer.withOptions(SanitizeHtmlHtmlSanitizer.paranoidOptions()) // shortcut to allow nothing
+htmlSanitizer.options.disallowedTagsMode = "discard"                   // discard tags (rather than escape them)
```

Comment:
Accessor to `options` doesn't exist, please remove these documentation lines.

Reply:
Updated instead to demonstrate current way of customising an existing set of options.